### PR TITLE
Sphereon support

### PR DESCRIPTION
**Pull request by sanderPostma**
Fixes #

## Proposed Changes
  - Verifiable presentation support for service SD
  - Gaia-X compliance filter in the Catalog  
image
(Filter item is stored at publish time in ddo.metadata.additionalInformation.compliance)
  
  - Gaia-X compliance indicator in "VerifiedBadge" component    
image

  - Gaia-X legal name lookup for "Publisher" component  
image

 - Gaia-X legal name lookup for catalog stored in ddo.metadata.additionalInformation.gaiaXInformation.serviceSD.verifiedPublisherName 
image
  
  - Self-service description collapse/expand option
image

  - Simple docker support (dev-mode)

## PS
I had to change my address.config.js to make it work.
```  whitelists: {
    'nft.owner': [
      '0xa98A6eefbAE870b88a9C7A43f4b50066A01c93b6',
      '0xDa4fc9E82Ac4E44207a1f74137493D3437D80761',
      '0xa98A6eefbAE870b88a9C7A43f4b50066A01c93b6'
    ],
    'datatokens.address': [
      '0x0995527d3473b3a98c471f1ed8787acd77fbf009',
      '0x985d3E0134bCE4d82b90d639d5d83627DE687870'
    ]
  },
```
The default config file gives me "Could not retrieve asset"

Also, the extra button in the filter panel disturbs the rendering of the sort panel. I mitigated the issues reducing the minimum width of the filter buttons and modified the overflow-x / overflow-y values in the .sortList css.

### CODE REVIEW

**Comment by sanderPostma** [comment]
I have made everything configurable now using the following .env vars:
```
NEXT_PUBLIC_BADGE_COMPLIANCE_LABELS_ENABLE=true
NEXT_PUBLIC_SERVICE_SD_COMPLIANCE_GAIA_X_ENABLE=false
NEXT_PUBLIC_CATALOG_FILTER_COMPLIANCE_GAIA_X_ENABLE=false
NEXT_PUBLIC_COMPLIANCE_TYPE_GAIA_X_CAPTION=Gaia-X compliant

NEXT_PUBLIC_SERVICE_SD_COMPLIANCE_FMDM_ENABLE=true
NEXT_PUBLIC_CATALOG_FILTER_COMPLIANCE_FMDM_ENABLE=true
NEXT_PUBLIC_COMPLIANCE_TYPE_FMDM_CAPTION=FMDM compliant
```

File: src/components/@shared/SDVisualizer/index.module.css
Comment (moritzkirstein) [inline src/components/@shared/SDVisualizer/index.module.css:1]: issue: @LucaMilanese90 can we help with refactoring this to use the css variables we set up?

File: src/components/@shared/VerifiedBadge/index.tsx [hunk 1]
```diff
@@ -48,6 +55,32 @@ export default function VerifiedBadge({
...
+                      key={complianceCaption}
+                      style={{ display: 'flex', alignItems: 'center' }}
+                    >
+                      <Checkmark
+                        style={{
+                          width: '24px',
+                          height: '24px',
+                          marginRight: '0.1rem'
+                        }}
+                      />
```
Comment (moritzkirstein) [inline src/components/@shared/VerifiedBadge/index.tsx:76]: issue: would prefer styles to be handled by css module file, to be in line with the rest of the portal code

File: src/components/Asset/AssetActions/Compute/FormComputeDataset.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ import Alert from '@shared/atoms/Alert'
 import { getTokenBalanceFromSymbol } from '@utils/web3'
 import { MAX_DECIMALS } from '@utils/constants'
 import Decimal from 'decimal.js'
+import { verifyRawServiceSD } from '@components/Publish/_utils'
```
Comment (moritzkirstein) [inline src/components/Asset/AssetActions/Compute/FormComputeDataset.tsx:19]: issue: unused import added here

**Review by moritzkirstein — COMMENTED** [review]
The `Gaia-X Compliance` filter does not show up for me. I tried to set all provided env variables to `"true"` just to test this, but can't get the filter to show on the catalogue page.

Added a few minor comments / change requests to help with cross-team maintainability

File: src/components/@shared/VerifiedBadge/index.tsx (hunk 1 above)
Comment (sanderPostma) [reply src/components/@shared/VerifiedBadge/index.tsx]: @moritzkirstein 
Did you pull the latest changes and restarted?
I just tried it on a clean repo clone and it just works. I can set in my .env
```
NEXT_PUBLIC_CATALOG_FILTER_COMPLIANCE_GAIA_X_ENABLE=true
```
(Not NEXT_PUBLIC_SERVICE_SD_COMPLIANCE_GAIA_X_ENABLE, that is to mark the asset as compliant during publishing.)
A few moments later the page enables the filter, I don't need to reload the page.

File: src/components/Asset/AssetActions/Compute/FormComputeDataset.tsx (hunk 2 above)
Comment (sanderPostma) [reply src/components/Asset/AssetActions/Compute/FormComputeDataset.tsx]: I saw Luca already fixed this.

File: src/components/@shared/VerifiedBadge/index.tsx (hunk 1 above)
Comment (moritzkirstein) [reply src/components/@shared/VerifiedBadge/index.tsx]: Okay thanks for the input, I will check this again.

**Review by moritzkirstein — APPROVED** [review]

**Pull request merged**